providers/logzio/logzio_provider.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2018 The Terraformer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logzio
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+
+	"github.com/pkg/errors"
+)
+
+type LogzioProvider struct {
+	terraform_utils.Provider
+	token   string
+	baseURL string
+}
+
+var (
+	disallowedChars = regexp.MustCompile(`[^A-Za-z0-9-]`)
+)
+
+const logzioProviderVersion = "~>v1.1.1"
+
+func (p LogzioProvider) GetResourceConnections() map[string]map[string][]string {
+	return map[string]map[string][]string{
+		"alert": {"endpoint": []string{"alert_notification_endpoints", "id"}},
```

Comment:
change to => `"alerts": {"alert_notification_endpoints": []string{"alert_notification_endpoints", "id"}},`

Reply:
Done, thank you very much! 🙏